### ISSUE TITLE
F1 - Debug, Breakpoint, Conditions, Actions?

### ISSUE THREAD

**Issue by nurbles** [MicrosoftDocs/visualstudio-docs#645]
I'm trying to find information about the new (to me, at least) ACTIONS available for conditional breakpoints in Visual Studio 2017 Pro.  I know how to see a variable's value, but I'm wondering if I can specify the value FORMAT (e.g. like I can in the Watch windows).  I landed here when I pressed F1 in te ACTIONS input field instead of going somewhere relevant and useful.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: 9a28ee58-bd99-e74a-0c0c-30f5a41da08e
* Version Independent ID: 2467c672-a567-583c-2801-87dcaf0f2c91
* [Content](https://docs.microsoft.com/en-us/visualstudio/ide/not-in-toc/default?f1url=https%3A%2F%2Fmsdn.microsoft.com%2Fquery%2Fdev15.query%3FappId%3DDev15IDEF1%26l%3DEN-US%26k%3Dk(VS.TextEditor)%26rd%3Dtrue#help-us-improve-f1)
* [Content Source](https://github.com/Microsoft/visualstudio-docs/blob/master/docs/ide/not-in-toc/default.md)
* Service: unspecified

**Comment by Mikejo5000** [comment]
@nurbles We have some information on this topic here: https://docs.microsoft.com/en-us/visualstudio/debugger/using-breakpoints#BKMK_Print_to_the_Output_window_with_tracepoints

**Comment by Mikejo5000** [comment]
I don't think the format specifiers are supported, however. I'll ask about it.

**Comment by nurbles** [comment]
Thanks for the link, which was the most detailed 12 sentences we had found about this new feature, hence my question here.  :-)
FWIW, we have discovered that appending ',o' and ',x' will display the value in octal or hex using the number of digits necessary to include all bits of the value's data type -- even if they are all zero.  So tracing <{Address,o}> or <{Address,x}> would display the Address value in octal or hex.  I apparently had entered a period instead of a dot when I first tried to use these and the font used in the trace dialog (coupled with my 17" screen) was too small to see the difference.  I am still interested in any other formatting options that may exist or other non-obvious features of this great new debugging tool.
